src/styles/header.scss

```diff
@@ -0,0 +1,24 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 40px;
+  height: 60px;
```

Comment:
don't hardcode height like this, use paddings in links to dictate the height of the header